packages/api/src/router/post.ts

```diff
@@ -0,0 +1,9 @@
+import { createTRPCRouter, publicProcedure } from "../trpc";
+
+export const postRouter = createTRPCRouter({
+  getPosts: publicProcedure.query(async ({ ctx }) => {
```

Comment:
Since this procedure will be authenticated (eventually), use protectedProcedure instead of public (can be imported from the same file)